src/main/java/com/spotinst/sdkjava/model/service/admin/organization/SpotAdminOrganizationService.java

```diff
@@ -0,0 +1,395 @@
+package com.spotinst.sdkjava.model.service.admin.organization;
+
+import com.spotinst.sdkjava.client.response.BaseSpotinstService;
+import com.spotinst.sdkjava.client.rest.*;
+import com.spotinst.sdkjava.exception.SpotinstHttpException;
+import com.spotinst.sdkjava.model.api.admin.organization.*;
+import com.spotinst.sdkjava.model.responses.admin.organization.*;
+import org.apache.http.HttpStatus;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+
+public class SpotAdminOrganizationService extends BaseSpotinstService {
```

Comment:
GetAccountMappingOfUser - https://api.spotinst.io/setup/accountUserMapping - this API is not implemented.